persist/src/main/resources/db/migration/V3_11__AddParentAccount.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS parent_accounts (
+    parent_id     INT     NOT NULL,
+    child_id      INT     NOT NULL,
+
```

Comment:
🔧 We should have a primary key here too, can be just `PRIMARY KEY (parent_id, child_id)`

Reply:
Not a SQL expert but does this not deserve its own autoincrementing id?

Reply:
It could, but the `parent_id` and `child_id` combination should be unique and is more meaningful than an autoincrementing id